docs/source/concepts.rst

```diff
@@ -891,7 +891,7 @@ Adaptive Micro-Batching
 
 .. note::
   The micro-batching option has become the default behavior starting release 0.12.0.
-  Use the --disable-microbatch option to turn off batching behavior.
+  Set --mb-max-batch-size=1 to turn off batching behavior.
```

Comment:
cc @bojiang  do we have any existing integration tests covering the behavior of `--mb-max-batch-size=1`?

Reply:
now we have https://github.com/bentoml/BentoML/pull/1661